lib/govuk_index/presenters/common_fields_presenter.rb

```diff
@@ -81,6 +81,18 @@ def section_id
       @_section_id ||= payload.dig("details", "section_id") if format == "hmrc_manual_section"
     end
 
+    def is_historic
+      is_political && payload.dig("expanded_links", "government", 0, "details", "current") == false
+    end
+
+    def is_political
+      payload.dig("details", "political") || false
```

Comment:
I'm assuming that the use of ` || false` here means that payload["details"]["political"] isn't always set?

Reply:
Yes, `dig` returns `nil` if any of the keys are not present.  I'm working on an assumption that some applications may send a payload that doesn't contain a political status.

Reply:
👍